### ISSUE TITLE
Would be possible to add a folder pannel to wxformbuilder please...

### ISSUE THREAD

**Issue by redcat22** [wxFormBuilder/wxFormBuilder#504]

**Comment by jhasse** [comment]
Do you mean wxDirDialog?

**Comment by Alchemist45** [comment]
Not really.  I am thinking about the functionality of the rpanel addon module in the R language.  It pops up a window that allows interactive options with a graph such as a moving cursor with cross hair readout.

 

Just asking.

 

Thanks,

 

Scott
